src/Simplex/Chat/Store/Direct.hs

```diff
@@ -424,6 +425,19 @@ updatePCCIncognito db User {userId} conn customUserProfileId = do
     (customUserProfileId, updatedAt, userId, pccConnId conn)
   pure (conn :: PendingContactConnection) {customUserProfileId, updatedAt}
 
+updatePendingContactConnectionUserId :: DB.Connection -> UserId -> UserId -> PendingContactConnection -> IO PendingContactConnection
+updatePendingContactConnectionUserId db userId oldUserId conn = do
```

Comment:
we try to shorten names where possible - there's no consistent rule, but method above uses "PCC"
```suggestion
updatePCCUserId :: DB.Connection -> UserId -> UserId -> PendingContactConnection -> IO PendingContactConnection
updatePCCUserId db userId oldUserId conn = do
```